lang/en.json

```
@@ -177,6 +179,10 @@
   "COMPLETE": "COMPLETE",
   "Start app tour to know talawa functioning": "Start app tour to know talawa functioning",
   "Scan QR": "Scan QR",
+  "The Camera is not working": "The Camera is not working",
```

Biome (2.1.2)

[error] The key The Camera is not working was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)